### ISSUE TITLE
README incorrect + maintenance ?

### ISSUE THREAD

**Issue by AdrienLemaire** [theatlantic/django-chosen#1]
Hi

Your README is wrong, you copied it from your python-blogger project and didn't update it correctly

By the way, the proper format to install from pip is pip install -e git+git://github.com/theatlantic/django-chosen.git#egg=django-chosen, that can be added in a requirements.txt file. As you already created a setup.py, you could also upload it on the pip packages.

Other point, you didn't commit anything since August, so I'm wondering if this project is still maintained ?

Thank you very much for your answers, and for your hard work on this project

**Comment by fdintino** [comment]
Sorry for the delay in getting back to you, we recently finished an exhausting development cycle.

I've fixed the issues in the README file. I left a comment on the other ticket about your pull request. We'll probably want to maintain a branch for the media changes. It won't take much time, but I probably won't have bandwidth until later this week to do that, at which point I'll merge in your pull request. Thanks for your interest and contribution to the project!
